src/stories/Blocks/page/Page.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC } from "react";
+import Hero, { HeroProps } from "../../Library/hero/Hero";
+
+type PageProps = {
+  hero?: HeroProps;
+};
+
+const Page: FC<PageProps> = ({ hero }) => {
+  return (
+    <article>
+      {hero && (
+        <Hero
```

Comment:
Could this hero be used for the eventpage as well? (eventHeader). 

Just a thought, it might be best to keep them seperate.

Reply:
Unfortunately not. they are too different